sql/src/main/java/io/crate/operation/reference/information/InformationColumnsExpression.java

```diff
@@ -46,6 +69,15 @@ public BytesRef value() {
         }
     }
 
+    public static class ColumnsTableCatalogExpression extends InformationColumnsExpression<BytesRef> {
+
+        @Override
+        public BytesRef value() {
+            assert row.info.ident().tableIdent().schema() != null : "table schema can't be null";
+            return new BytesRef(row.info.ident().tableIdent().name());
```

Comment:
this returns the table name, which is wrong. use the schema expression instance. pls document that this gives back the schema